src/PVRIptvData.cpp

```diff
@@ -469,12 +470,23 @@ bool PVRIptvData::LoadPlayList(void)
         tmpChannel.properties.insert({prop, propValue});
       }
     }
+    else if (StringUtils::Left(strLine, (int)strlen(PLAYLIST_TYPE_MARKER)) == PLAYLIST_TYPE_MARKER)
+    {
+      std::string value = ReadMarkerValue(strLine, PLAYLIST_TYPE_MARKER);
+      if (value == "EVENT")
+        tmpChannel.properties.insert({PVR_STREAM_PROPERTY_ISREALTIMESTREAM, "true"});
+      else if (value == "VOD")
+        tmpChannel.properties.insert({PVR_STREAM_PROPERTY_ISREALTIMESTREAM, "false"});
+    }
     else if (strLine[0] != '#')
     {
       XBMC->Log(LOG_DEBUG,
                 "Found URL: '%s' (current channel name: '%s')",
                 strLine.c_str(), tmpChannel.strChannelName.c_str());
 
+      if (tmpChannel.properties.find(PVR_STREAM_PROPERTY_ISREALTIMESTREAM) == tmpChannel.properties.end())
+        tmpChannel.properties.insert({PVR_STREAM_PROPERTY_ISREALTIMESTREAM, "true"});
```

Comment:
Interesting approach, but couldn't this be coded much simpler, without searching the prop?

Reply:
I would simple define and fill a bool bIsRealTime and if it is true I would append the prop after the while loop.